internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -850,6 +850,19 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Filesystem'
+        services:
+          type: object
+          properties:
+            enabled:
```

Comment:
a small `description` would be nice I think

Reply:
Done